config/puma/production.rb

```diff
@@ -13,6 +13,10 @@
 #
 workers ENV.fetch('FOREMAN_PUMA_WORKERS', 2).to_i
 
+# Provides systemd notify support through the
+# puma-systemd-plugin
+plugin :systemd
```

Comment:
Does this raise an exception if it can't be loaded?

Reply:
Aye, I am catching that now.